client_plugin/drivers/shared/shared_driver.go

```diff
@@ -234,6 +234,7 @@ func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 	volRecord := VolumeMetadata{
 		Status:         kvstore.VolStateCreating,
 		GlobalRefcount: 0,
+		Port:           1445,
```

Comment:
This is just for making this partial part working.
This should be removed after your PR is in.